app/controllers/performance_tracking_controller.rb

```diff
@@ -5,8 +5,11 @@ def index; end
 
   private
 
+  # Users that can manage others are only allowed to access this page on staging.
+  # This is configured by the allow_user_managers_service_access feature flag.
   def require_supervisor!
```

Comment:
I would remove this comment as the `FeatureFlags.allow...` line of code says the same thing imo.

Reply:
We'll could adjust the comment to explain why user managers can have access in staging (i.e. for testing purposes). I agree the second line isn't necessary